benchmarks/setup.rb

```diff
@@ -39,7 +39,7 @@ def tags
 
 def users_with_combined_tasks
   @users_with_combined_tasks ||= rom.relation(:users).as(:user_with_combined_tasks)
-    .combine(rom.relation(:tasks).for_users)
+                                 .combine(rom.relation(:tasks).for_users)
```

Comment:
Add a commit to this one please

On Tue, May 26, 2015 at 2:57 PM Errin Larsen <notifications@github.com>
wrote:

> In benchmarks/setup.rb
> <https://github.com/rom-rb/rom/pull/240#discussion_r31029638>:
>
> > @@ -39,7 +39,7 @@ def tags
> >
> >  def users_with_combined_tasks
> >    @users_with_combined_tasks ||= rom.relation(:users).as(:user_with_combined_tasks)
> > -    .combine(rom.relation(:tasks).for_users)
> > +                                 .combine(rom.relation(:tasks).for_users)
>
> I can fix this, and update the rubocop config to stop reporting these
> offenses.
>
> Want me to create a new pull-request? add a commit to this one?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rom-rb/rom/pull/240/files#r31029638>.
>